src/components/common/windowListener.js

```diff
@@ -2,14 +2,19 @@ export class WindowListener {
   constructor (callback) {
     this.callback = callback
     this.elements = {}
```

Comment:
I would suggest that elements is not a good name for this variable. Look for something more accurate.